sdk/src/connections/network/network_depth_sensor.cpp

```diff
@@ -303,6 +303,13 @@ aditof::Status NetworkDepthSensor::program(const uint8_t *firmware,
                                            size_t size) {
     using namespace aditof;
 
+    if (firmware == nullptr) {
+        LOG(ERROR) << "Received firmware null pointer";
+        return Status::INVALID_ARGUMENT;
+    }
+
+    assert(size > 0);
```

Comment:
`size` has the type `size_t` which by definition is unsigned, so this is not needed.